ddtrace/tracer/tracer_go11.go

```diff
@@ -0,0 +1,16 @@
+// +build go1.11
+
+package tracer
+
+import (
+	"context"
+	t "runtime/trace"
+)
+
+func startExecutionTracerTask(name string) func() {
+	if !t.IsEnabled() {
+		return func() {}
+	}
+	_, task := t.NewTask(context.TODO(), name)
```

Comment:
I'd flip a coin on this about whether to use `Background` vs `TODO`.
The difference comes down to intent: "starting fresh from here" vs "will probably link to something else in the future".

Reply:
Yep, exactly the latter is what is meant here.